### PR TITLE
Login gui

### PR DESCRIPTION
**Pull request by Anders-Bacon**
Frontend part of Login and Create account page.
The pages are currently routing to the welcomepage, because their correct routes dont exists yet.
The css is rensposive and works on all devices.

### CODE REVIEW

**Review by MidnightRocket — CHANGES_REQUESTED** [review]
I think that files in "resources/users/" should be moved to "resources/account" for account related stuff. 
I also think that the url endpoint "/newAcc" should be "/signup" to better meet standards.

File: resources/views/users/newAcc.blade.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                <input id="password" name="password" type="password"> <br />
+                <label>Confirm password:</label> <br />
+                <input id="password" name="password" type="password"> <br /><br />
+                <button type="submit" class="SigninBtn">Create Account</button>
+            </form>
+        </article>
+    </div>
+
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js" integrity="sha384-cVKIPhGWiC2Al4u+LWgxfKTRIcfu0JTxR+EQDz/bgldoEyl4H0zUF0QKbrJ0EcQF" crossorigin="anonymous"></script>
```
Comment (MidnightRocket) [inline resources/views/users/newAcc.blade.php:34]: I think these script tags should be moved into the html head. And then use the `defer` attribute to defer execution until document load.

**Review by MidnightRocket — CHANGES_REQUESTED** [review]

File: resources/views/users/newAcc.blade.php (hunk 1 above)
Comment (Anders-Bacon) [reply resources/views/users/newAcc.blade.php:34]: Everything should be fixed to the feedback recieved. the scripts are now in the head, with a defer attribute and every path, and pathname have been fixed as well.

File: resources/views/users/newAcc.blade.php
Comment (MidnightRocket) [inline resources/views/users/newAcc.blade.php:1]: What about these two files in users?

**Review by MidnightRocket — APPROVED** [review]

**Pull request merged**